### DISCUSSION TITLE
Github has been flagged my account. Please can anyone help me to fix it?

### DISCUSSION THREAD

**Discussion by foysal619** [community/community#52689] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

My github has flagged automatically. I don't know why this happened. Please can anyone help me to reach the github support?

**Comment by cehfisher** [comment]
Hi @foysal619 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
To open a ticket, please use our [Support page](https://support.github.com/contact).
